packages/netlify/src/index.ts

```diff
@@ -173,6 +173,18 @@ export interface NetlifyIntegrationConfig {
 	 * @default {true}
 	 */
 	imageCDN?: boolean;
+
+	/**
+	 * Extra files to include in the SSR function bundle. Paths are relative to the project root.
+	 * Glob patterns are supported.
+	 */
+	includedFiles?: string[];
+
+	/**
+	 * Files to exclude from the SSR function bundle. Paths are relative to the project root.
+	 * Glob patterns are supported.
+	 */
+	excludedFiles?: string[];
```

Comment:
What was requested by the user is a way to include files right? Why would someone one to exclude some files if it's already the case by default?

Reply:
Two cases: either they've added a glob but want to exclude one specific file from that. This is a more intuitive way than using a negated glob. The other is to exlude automatically traced dependencies, like binaries in node moduels that it incorrectly thinks are needed. This could be used to exclude them.